src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -0,0 +1,276 @@
+        button.leftAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: -10).isActive = true
+        stagePresenterViewController.menuScreenshotButton = button
+
+        button = UIButton(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        self.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.rightAnchor.constraint(equalTo: self.rightAnchor).isActive = true
+        button.leftAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.topAnchor.constraint(equalTo: stagePresenterViewController.menuScreenshotButton.bottomAnchor, constant: 5).isActive = true
+        stagePresenterViewController.menuScreenshotLabel = button
+
+        //axis
+        button = UIButton(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        self.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.rightAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.leftAnchor.constraint(equalTo: self.leftAnchor).isActive = true
+        button.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -15).isActive = true
+        stagePresenterViewController.menuAxisLabel = button
+
+        button = UIButton(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        self.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.rightAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.leftAnchor.constraint(equalTo: self.leftAnchor).isActive = true
+        button.bottomAnchor.constraint(equalTo: self.stagePresenterViewController.menuAxisLabel.topAnchor, constant: -5).isActive = true
+        stagePresenterViewController.menuAxisButton = button
+
+        //aspectratio
+        button = UIButton(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        self.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.rightAnchor.constraint(equalTo: self.rightAnchor).isActive = true
+        button.leftAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -15).isActive = true
+        stagePresenterViewController.menuMaximizedMinimizedLabel = button
+
+        button = UIButton(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        self.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.rightAnchor.constraint(equalTo: self.rightAnchor).isActive = true
+        button.leftAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.bottomAnchor.constraint(equalTo: stagePresenterViewController.menuMaximizedMinimizedLabel.topAnchor, constant: -5).isActive = true
+        stagePresenterViewController.menuAspectRatioButton = button
```

Comment:
Is it possible to right one or a few different (if one would be to complicated) methods where you pass everything you need -> for example the type of button (i.e menuAspectRationButton), the constant (5, -15), the anchor (left, right) and different labels? I get if it's more work than benefit, but it seems that there could be a code reduction.